tools/SetupFlow/DevHome.SetupFlow/Controls/SetupShell.xaml

```diff
@@ -15,93 +15,82 @@
         <converters:BoolToObjectConverter x:Key="StepperBoolToGlyphBrushConverter" TrueValue="{ThemeResource AccentFillColorDisabledBrush}" FalseValue="{ThemeResource AccentFillColorDefaultBrush}" />
         <converters:BoolToVisibilityConverter x:Key="NegatedBoolToVisibilityConverter" TrueValue="Collapsed" FalseValue="Visible" />
     </UserControl.Resources>
-    <Grid RowSpacing="6" x:Name="ShellContent">
+    <Grid x:Name="ShellContent">
         <Grid.RowDefinitions>
-            <RowDefinition Height="auto"/>
-            <RowDefinition Height="auto"/>
             <RowDefinition Height="auto"/>
             <RowDefinition Height="*"/>
         </Grid.RowDefinitions>
-        <!-- Title -->
-        <StackPanel Grid.Row="0">
+
+        <StackPanel Spacing="6" Visibility="{x:Bind HeaderVisibility, Mode=OneWay}">

```

Comment:
If properties fit on a single line, we should consider keeping them on a single line to keep the file short. Otherwise, I suggest breaking them one per line. The same rule applies to C# code 😉